tools/isoplot/isoplot.xml

```diff
@@ -0,0 +1,154 @@
+<tool id='isoplot' name='Isoplot: Generate plots from isocor output' version='0.2.0' profile='20.01'>
+    <requirements>
+        <requirement type='package' version='1.3.0'>isoplot</requirement>
+    </requirements>
+    <command><![CDATA[
+         isoplot '$datafile' 'galaxy' '$format' --value '$value' --template '$template_file'
+#if $SP.generate_barplot:
+    -bp
+#end if
+#if $SP.generate_meanplot:
+    -mb
+#end if
+#if $SP.generate_heatmap:
+    -hm
+#end if
+#if $SP.generate_clustermap:
+    -cm
+#end if
+#if $IP.generate_barplot:
+    -IB
+#end if
+#if $IP.generate_meanplot:
+    -IM
+#end if
+#if $IP.generate_heatmap:
+    -HM
+#end if
+#if $stack:
+    -s
+#end if
+#if $verbosity:
+    -v
+#end if
+#if $annotation:
+    -a
+#end if
+-g -z '$plots_output' > $data_output 2> '$log_output'
+    ]]></command>
+    <inputs>
+        <param name='datafile' type='data' format='csv, tabular' multiple='true' label='Upload data file' />
+        <param name='template_file' type='data' format='csv, xlsx' multiple='true' label='Template file' argument='--template'/>
+        <!--param name='run_name' type='text' label='Run name' /-->
+        <param name='format' type='select' label='Plot format'>
+            <option value='pdf'>pdf</option>
+            <option value='jpeg'>jpeg</option>
+            <option value='png'>png</option>
+            <option value='svg'>svg</option>
+            <option value='html'>html</option>
+        </param>
+        <param name='value' type='select' label='Data to plot' >
+            <option value='corrected_area'>Corrected Area</option>
+            <option value='isotopologue_fraction'>Isotopologue Fraction</option>
+            <option value='mean_enrichment'>Mean Enrichment</option>
+        </param>
+        <section name='SP' title='Select static plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' truevalue='-bp' falsevalue=''/>
+            <param type='boolean' name='generate_meanplot' label='Flag to generate meaned barplots' truevalue='-mb' falsevalue=''/>
+            <!--<param type='boolean' name='generate_areaplot' label='Flag to generate stacked area plots'
+                   argument='-sa' optional='true'/>/-->
+            <param type='boolean' name='generate_heatmap' label='Flag to generate heatmap' truevalue='-hm' falsevalue=''/>
+            <param type='boolean' name='generate_clustermap' label='Flag to generate clustermap' truevalue='-cm' falsevalue=''/>
+        </section>
+        <section name='IP' title='Select interactive plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' truevalue='-IB' falsevalue=''/>

```

Comment:
Can you use `argument` in place of `name` if the param map exactly isoplot args?
Here: 
```xml
<param type='boolean' argument='--interactive_barplot' label='Flag to generate barplots' truevalue='-IB' falsevalue=''/>
```

You will be able to use `$SP.interactive_barplot` within the `<command>` section.